### ISSUE TITLE
confused with the conv layer

### ISSUE THREAD

**Issue by tc-yue** [guoyinwang/LEAM#6]
`Att_v = tf.contrib.layers.conv2d(G, num_outputs=opt.num_class,kernel_size=[opt.ngram], padding='SAME',activation_fn=tf.nn.relu) #b * s *  c`
The implementation code above is a conv2D operation on the match matrix G.
while the formulations in the paper below seems only one filter with the size(2r+1) to produce a further match matrix (K*L).  I think they are little different. Is it True?
```math
u = Relu(GW+b)  
```
where W ∈ R2r+1 and b ∈ R K   u l ∈ R K

**Comment by ReactiveCJ** [comment]
@tc-yue The code can not run well, the conv2d is not suitable when the input dim is three.

**Comment by ReactiveCJ** [comment]
@tc-yue it is just a conv1d so the size of W is (2r+1) * K * K

**Comment by lanjinraol** [comment]
@tc-yue Hello, I have same confusion with you. I think it's different between the two operations. In the paper, the filter with size(2r + 1)  only captures contextual feature, while in the implementation code the K filters capture contextual features and relationships among the categories. Have you gotten any reasonable explanation?
